src/app/shared/entity/datacenter.ts

```diff
@@ -20,7 +20,6 @@ export class CreateDatacenterModel {
 export class Datacenter {
   metadata: Metadata;
   spec: DatacenterSpec;
-  seed?: boolean;
```

Comment:
@lsviben Can you remove it from the backend too? It is not required anymore.

Reply:
hmm it should have been removed by this - https://github.com/kubermatic/kubermatic/pull/5562/files

Reply:
Okay then, thanks.